app/views/shared/_money_input.html.slim

```diff
@@ -1,7 +1,9 @@
+-label_size ||= "s"
```

Comment:
Sorry, I was wrong, the default _isn't_ "s". The size is now right, but this is still causing the label to be *bold* on all the other screens when previously it wasn't.